src/applications/hca/selectors.js

```diff
@@ -9,16 +9,31 @@ import { HCA_ENROLLMENT_STATUSES } from './constants';
 // top-level selectors
 export const selectEnrollmentStatus = state => state.hcaEnrollmentStatus;
 export const isEnrollmentStatusLoading = state =>
-  selectEnrollmentStatus(state).isLoading;
+  selectEnrollmentStatus(state).isLoadingApplicationStatus;
+export const isLoadingDismissedNotification = state =>
+  selectEnrollmentStatus(state).isLoadingDismissedNotification;
 export const hasServerError = state =>
   selectEnrollmentStatus(state).hasServerError;
 export const noESRRecordFound = state =>
   selectEnrollmentStatus(state).noESRRecordFound;
 export const isShowingHCAReapplyContent = state =>
   selectEnrollmentStatus(state).showHCAReapplyContent;
+export const isInESR = state => {
+  const status = selectEnrollmentStatus(state).enrollmentStatus;
+  return (
+    new Set([
```

Comment:
You could pull this Set instantiation out of the selector here, I think. Works either way, though.

Reply:
could also track `isInESR` on the state explicitly i.e. use a reducer.

Reply:
This one is killing me because it is no longer just checking to see if the user is in ESR. What this selector is really being used for is just a part of the logic to decide if we need to show the user their HCA status on the dashboard. And I can't think of a better name for it that doesn't get crazy verbose. I might change it with my next PR, but am totally open to ideas. Somehow `isInESRAndNotDeceasedOrDeclined` isn't cutting it for me.